pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm

```diff
@@ -290,10 +260,26 @@ sub _suggest_imunify360 {
 sub _suggest_iav {
     my ($self) = @_;
 
-    if ( !$self->{iav}{installed} && !$self->{iavp}{licensed} ) {
+    if ( $self->{iav}{installed} && !$self->{iavp}{licensed} ) {
```

Comment:
maybe easier to read by using a block `if ( !$self->{iavp}{licensed} ){ ....` but just one idea
the two if / else if could then be inside that block 
nothing wrong there

Reply:
done